docs/messaging.rst

```diff
@@ -1122,6 +1122,41 @@ Frontends can choose how they update prior outputs (or if they regard this as a
 regular ``display_data`` message). Within the jupyter and nteract_ notebooks,
 all displays that match the ``display_id`` are updated (even if there are multiple).
 
+Transient Display Data
+----------------------
+
+.. versionadded:: 5.3
+
+This message is very similar to `display_data` but its is content is meant to be
+transient (e.g. not saved in output areas of notebooks). Contents that can be sent
+through this message type can include but not limited to status information of long
+calculation and debug information.
+
+.. _transient_display_data:
+
+Message type: ``transient_display_data``::
+
+    content = {
+
+        # Description of the data
+        'title': string,
```

Comment:
If multiple messages come in with different titles, do they each occupy new areas?

Is this regarded as a unique identifier then?

Reply:
See my previous response. `title` is intended to group consecutive messages with the same `title` (and `append=True`), very similar to the aggregation of multiple `display_data` for the execution of the same `input`. It is possible to keep (short) history of transient messages (e.g. a navigation button) but it would be easier to start with a single widget and see if there is a strong need from users to keep some sort of history.

Reply:
Ok, this is definitely getting used as a unique id then. It's both a display name for the user and the way it groups. I want to clarify that here in the doc if this is what you mean.

Reply:
Will do.